### ISSUE TITLE
Phase out GI numbers

### ISSUE THREAD

**Issue by mlbendall** [gwcbi/PathoScope#2]
NCBI is phasing out GI numbers, PathoScope LIB needs to do so as well.

**Comment by ecastron** [comment]
@mlbendall we have a script that does that https://github.com/microgenomics/pasteTaxID. It looks for different types of identifiers and appends the taxid to fasta entries in a group of individual files or a multifasta file. It does so on the fly using Entrez tools so no need to download the taxonomy DB.

If nobody is on it yet, we could try to fix pathoLIB using a similar strategy. What do you think?

**Comment by mlbendall** [comment]
I'd say go for it! Just create a new branch and make a pull request.

Perhaps to maintain backwards compatibility, we can create another subcommand ("LIB2", "LIB+" or similar) that uses this approach. Then just call this script with subprocess.

**One possible snag**: This script relies heavily on linux commands (i.e. awk, sed) so is unlikely to work in Windows. The current version of PathoScope is (supposedly) cross-platform. I'm at peace with discontinuing windows support, but I think we should come to a consensus on this.

**Comment by ecastron** [comment]
I agree. We should maintain backwards compatibility and cross-platform support. @sanrrone and I will work on it
